softdev.bib

```diff
@@ -2571,3 +2571,14 @@ @inproceedings{trudel13really
     month = Jul,
     doi = "10.1007/978-3-642-39038-8_20"
 }
+
+@inproceedings{Carr2017,
```

Comment:
I think this file is nearly all 4-space indents, so let's try and do the same? [I know there are exceptions, but we probably shouldn't add to them :) ]